### PR TITLE
support attached COSE payloads and presentations

### LINKED ISSUE #31: +cose examples look wrong (diagnostic mode broken?)
The +cose examples that respec-vc spits out look wrong. The payload is `nil`, and if that's what is being signed, then all the +cose examples are wrong as well. Tagging @decentralgabe to look into fixing this (as we'll need this fixed before we go to PR).

### PR DESCRIPTION
**Pull request by thereisnogabe**
Fix #31 

Also added support for rendering presentation examples, I've only enabled this for vc-jose-cose work for now.

There is also #30 which I believe should follow this once media types settle down.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -81,6 +81,40 @@ <h1>Basic Example</h1>
...
+  <pre class="example nohighlight vc" data-vc-tabs="jose sd-jwt cose" title="A Verifiable Presentation example">
+{
+  <span class='comment'>// set the context, which establishes the special terms we will be using.</span>
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  <span class='comment'>// specify the identifier for the presentation</span>
+  "id": "http://example.edu/presentations/1872",
+  <span class='comment'>// the presentation type, which declare what data to expect in the presentation</span>
```
Comment (TallTed) [inline index.html]:
```suggestion
  <span class='comment'>// the presentation type, which declares what data to expect in the presentation</span>
```

**Review by TallTed — COMMENTED** [review]
Media types in this appear to be incorrect (as of this writing), as many have multiple plus signs, which have again been declared invalid by IETF.

**Comment by thereisnogabe** [comment]
@TallTed yes we're in an awkward place here since no media types have (yet) been approved by the IETF

I should update this PR (will shortly) to at least align with vc-jose-cose as of [this PR](https://github.com/w3c/vc-jose-cose/pull/279).

**Comment by thereisnogabe** [comment]
nudge @msporny hope we can get this in and then adjust the media types when things settle

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Merged and released v3.3.0 -- can you please check to make sure it does what you expect it to, @decentralgabe? I tested it locally on the test document you provided and it seemed to work for me.

**Comment by thereisnogabe** [comment]
lgtm thanks @msporny

**Pull request merged**